xbmc/cores/VideoPlayer/VideoPlayerVideo.cpp

```diff
@@ -61,6 +61,7 @@ CVideoPlayerVideo::CVideoPlayerVideo(CDVDClock* pClock
 
   m_bRenderSubs = false;
   m_stalled = false;
+  m_stillframe = false;
```

Comment:
this member is pretty useless because its information is exactly the same as m_stalled.

Reply:
Not necessarily; m_stalled can also go to true at these two locations which aren't necessarily the same as having a still frame, at least not the first one, correct?

https://github.com/xbmc/xbmc/blob/fbef8cf4aca97418eb04f231a973ed4e4945c164/xbmc/cores/VideoPlayer/VideoPlayerVideo.cpp#L218

https://github.com/xbmc/xbmc/blob/fbef8cf4aca97418eb04f231a973ed4e4945c164/xbmc/cores/VideoPlayer/VideoPlayerVideo.cpp#L459

Perhaps I'm being obtuse and it doesn't matter, but not having any packets during OpenStream() seems like a different condition.  It behaves differently for this particular case, at least.  There are no video packets when you open these, so it starts as stalled, but not necessarily due to a still frame (yet), there simply is no data.

Reply:
trust me, you change won't distinguish between stalled and real still frame. apart from that you introduce issues by not resetting m_stillframe i.e. on flush. with that you create a condition that m_stillframe won't get reset anymore. but it doesn't matter anyway because this approach is wrong.